test/test_base_alloc_linear.cpp

```diff
@@ -77,6 +81,10 @@ TEST_F(test, baseAllocLinearMultiThreadedAllocMemset) {
                 UT_ASSERTeq(*(buffer[i].ptr + k), (i + TID) & 0xFF);
             }
         }
+
+        for (int i = 0; i < ITERATIONS; i++) {
```

Comment:
nit: it would be good to also have a test that frees the pointers in a different order (e.g random + from last to first).

Reply:
Yes, it would be good. BTW random and real-life scenario is already tested by the proxy library.

Reply:
@bratpiorka I think it can be a separate task in Jira and we can give it someone else.

Reply:
done